.github/workflows/commitlint.yaml

```
@@ -8,6 +8,7 @@ on:
 jobs:
   commitlint:
     name: commitlint
+    if: ${{ github.actor != 'app/dependabot' }}
```

```suggestion
    if: ${{ github.actor != 'dependabot[bot]' }}
```
@nixpanic Can add reference to where you found example for `'app/dependabot'`. 
This website mentions it as `'dependabot[bot]'`
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions

---

I was wondering about that too!

https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context mentions for `github.actor`:

> The login of the user that initiated the workflow run.

And that seems to be `app/dependabot` according to the PRs that the bot created.

The link you gave has more examples specifically for dependabot, so I'd trust that more too. Will change it!